css/index.css

```diff
@@ -1,3 +1,480 @@
+html {
+	margin: 0 auto;
+	padding: 0;
+}
+body {
+	width: 876px;
+	margin: 0 auto;
+	padding: 0;
+ 	
+ }
+ header {
+	position: fixed;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	height: 60px;
+	width: 876px;
+	margin-bottom: 40px;
+	align-items: center;
+	background: rgb(101, 80, 109);
+	
+	
+ }
+ header>nav {
+	width: 60%;
+	height: 100%;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+ header>nav>a {
+	width: 20%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	color: black;
+	text-decoration: none;
+}
+ header>nav>a:hover {
+	background: black;
+	color: white;
+	border-bottom: 2px solid gray;
+	font-weight: bold;
+}
+ header>img {
+	display: flex;
+	justify-content: center;
+	width: 30%;
+	height: 100%;
+	max-height: 40px;
+	max-width: 100px;
+	margin-right: 40px;
+}
+ .get-started {
+	padding-top: 60px;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-around;
+	align-items: center;
+	height: 400px;
+	width: 100%;
+	border-bottom: 1px solid black;
+}
+ .get-started>section {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	height: 100%;
+	width: 50%;
+	order: 1;
+}
+ .get-started>section>h1 {
+	font-size: 3em;
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+}
+ .get-started>section>a, .grid>a {
+	text-decoration: none;
+	color: black;
+	border: 1px solid black;
+	border-left: 2px solid black;
+	border-bottom: 2px solid black;
+	padding: 5px 30px;
+}
+ .get-started>img {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	order: 2;
+	max-height: 280px;
+	height: 100%;
+	max-width: 280px;
+	width:100%;
+}
+ .main {
+	height: 500px;
+	width: 100%;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	border-bottom: 1px solid black;
+	justify-content: space-between;
+	margin-bottom: 40px;
+	padding: 0;
+}
+ #features, #about {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	height: 140px;
+	width: 44%;
+	justify-content: center;
+	
+	margin: 0;
+	padding: 0;
+	
+	
+}
+ #product, #vision, #services {
+	display: flex;
+	width: 33%;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: center;
+	margin: 0;
+	padding: 0;
+}
+ .middle-img {
+	margin-bottom: 40px;
+}
+ #contact {
+	
+	margin-bottom: 40px;
+}
+ footer {
+	display: flex;
+	justify-content: center;
+}
+ /* services page */
+ #services-services {
+	display: flex;
+    flex-direction: column;
+    padding-top: 20px;
+	margin-top: 40px;
+	margin-bottom: 40px;
+	border-bottom: 1px solid black;
+	height: 120px;
+}
+ #services-services>h1 {
+	font-size: 1.5em;
+}
+ .main-section {
+	width: 100%;
+	height: 660px;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	border-bottom: 1px solid black;
+	justify-content: space-between;
+	margin-bottom: 40px;
+}
+ .grid {
+	display: flex;
+	flex-direction: column;
+	width: 48%;
+	height: 200px;
```

Comment:
The buttons overflow outside of the grey box in tablet view. Consider removing the height and adding a `margin-bottom: 20px`